aptos-move/framework/src/natives/cryptography/groups/blst_backend.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Aptos
```

Comment:
Have you decided between `blst` and `arkworks` yet?

Cause keeping two implementations around will be extra work...